collection-scripts/gather

```diff
@@ -78,38 +76,26 @@ case "$CLUSTER" in
     "HUB")
 
     check_managed_clusters
-    #oc adm inspect  ns/open-cluster-management  --dest-dir=must-gather
-    oc get pods -n "$HUBNAMESPACE" > ${BASE_COLLECTION_PATH}/gather-acm.log
-    oc get csv -n "$HUBNAMESPACE" > ${BASE_COLLECTION_PATH}/gather-acm.log 
-    oc adm inspect  ns/"$HUBNAMESPACE"  --dest-dir=must-gather
-    oc adm inspect  ns/open-cluster-management-hub  --dest-dir=must-gather
+    oc get pods -n "backplane-operator-system" > ${BASE_COLLECTION_PATH}/gather-acm.log
+    oc get csv -n "backplane-operator-system" > ${BASE_COLLECTION_PATH}/gather-acm.log 
+    oc adm inspect  ns/"backplane-operator-system"  --dest-dir=must-gather
+    oc adm inspect  ns/backplane-operator-system  --dest-dir=must-gather
```

Comment:
It looks like this is repeating the same call as the line before it. I believe we still deploy resources to the open-cluster-management-hub namespace so we still want to gather from it